app-modules/form/src/Filament/Blocks/Select.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Assist\Form\Filament\Blocks;
+
+use Assist\Form\Models\FormItem;
+use Filament\Forms\Components\KeyValue;
+use Filament\Forms\Components\Select as FilamentSelect;
+
+class Select extends Block
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->label('Select');
+    }
+
+    public static function make(string $name = 'select'): static
+    {
+        return parent::make($name);
+    }
```

Comment:
I can't make this abstract because it's overriding the base class. Instead I made an abstract method called type, and moved the make call into the base class which calls type. I can't call it name or key because these are also overriding the base class.